src/components/RoomNameInput.js

```diff
@@ -119,17 +126,24 @@ class RoomNameInput extends Component {
 
     render() {
         return (
-            <TextInputWithPrefix
-                disabled={this.props.disabled}
-                label={this.props.translate('newRoomPage.roomName')}
-                prefixCharacter="#"
-                placeholder={this.props.translate('newRoomPage.social')}
-                containerStyles={[styles.mb5]}
-                onChangeText={roomName => this.checkAndModifyRoomName(roomName)}
-                value={this.state.roomName.substring(1)}
-                errorText={this.state.error}
-                autoCapitalize="none"
-            />
+            <>
+                <TextInputWithPrefix
+                    disabled={this.props.disabled}
+                    label={this.props.translate('newRoomPage.roomName')}
+                    prefixCharacter="#"
+                    placeholder={this.props.translate('newRoomPage.social')}
+                    containerStyles={[styles.mb5]}
+                    onChangeText={roomName => this.checkAndModifyRoomName(roomName)}
+                    value={this.state.roomName.substring(1)}
+                    errorText={this.state.error}
```

Comment:
NAB since the input functionality is unrelated to these changes. It's really weird that we pass `errorText` here, but then [omit](https://github.com/Expensify/App/blob/e9c5e02d043420f1c447b1b1b418cf31e903124b/src/components/TextInputWithPrefix/index.js#L32) passing it on to `TextInput` and have to use `InlineErrorText` to display the error. Not sure why this is the case @jasperhuangg

Reply:
OH good catch! Honestly maybe it's better to just pass it in? Maybe there was a weird styling reason why we didn't do it. Let me try modifying that omit and seeing if that makes it better.

Reply:
Given how `TextInputWithPrefix` currently works, we will have to pass it here so the outline border changes to red. I'm not sure why we omit it though, but I assume it's intentional/necessary since it was added.

Reply:
Hmm removing the omit doesn't work - no error text shows up. Maybe it has to do with us using the [base ReactNative TextInput there in TextInputWithPrefix/index.js](https://github.com/Expensify/App/blob/d6a607324f84e654da2da6e073b150a00def7490/src/components/TextInputWithPrefix/index.js#L3) and it doesn't seem to use the errorText correctly. I think I'll still move around the error to TextInputWithPrefix and out of TextInputWithLabel (I don't think this is used anymore)